examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts

```diff
@@ -42,13 +42,13 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
 
   async getAllBpiSubjectAccounts(): Promise<BpiSubjectAccount[]> {
     const bpiSubjectAccountsModels = await this.bpiSubjectAccount.findMany({
-      include: { OwnerBpiSubject: true, CreatorBpiSubject: true },
+      include: { ownerBpiSubject: true, creatorBpiSubject: true },
     });
     return bpiSubjectAccountsModels.map((bp) => {
       return new BpiSubjectAccount(
```

Comment:
now that the prisma model and domain model are in sync, we can directly invoke mapper(bp, BpiSubjectAccount,BpiSubjectAccount)